packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -107,17 +116,20 @@ class _BottomAppBarState extends State<BottomAppBar> {
 
   @override
   Widget build(BuildContext context) {
-    final CustomClipper<Path> clipper = widget.shape != null
+    final ThemeData theme = Theme.of(context);
```

Comment:
I don't think you use this theme variable.

Reply:
Oh actually it is used below for color:

`color: widget.color ?? babTheme.color ?? theme.bottomAppBarColor,`

Reply:
But, it is only used once, and typically our pattern is if the theme variable is only used once, to inline the `Theme.of(context)` call! So I shouldn't be making this variable really.

Reply:
Fixed in https://github.com/flutter/flutter/pull/26765